src/Fixer/Comment/NoTrailingWhitespaceInCommentFixer.php

```diff
@@ -53,14 +53,14 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
         foreach ($tokens as $index => $token) {
             if ($token->isGivenKind(T_DOC_COMMENT)) {
-                $tokens[$index] = new Token([T_DOC_COMMENT, Preg::replace('/[ \t]+$/m', '', $token->getContent())]);
+                $tokens[$index] = new Token([T_DOC_COMMENT, Preg::replace('/(*ANY)[ \t]+$/m', '', $token->getContent())]);
```

Comment:
`(*ANY)` ? what is it ?

Reply:
> all Unicode newline sequences

https://www.pcre.org/original/doc/html/pcrepattern.html

Reply:
Uau, `Today I Learned`